src/person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    return true if of_age? || @parent_permission == true
```

Comment:
- [ ] You do not need return here, since the last thing is returned always. So I would suggest you to update this line to:
```ruby
  def can_use_services? 
    is_of_age?() || @parent_permission
  end
```